code/web/src/modules/user/api/state.test.js

```diff
@@ -0,0 +1,70 @@
+import '@testing-library/jest-dom'
+import state from './state'
+import { SET_USER, LOGIN_REQUEST, LOGIN_RESPONSE, LOGOUT } from './actions'
```

Comment:
These look great as well. Fairly simple but it for sure fleshes out the testing.